### DISCUSSION TITLE
为什么不直接用byte-level的bpe tokenizer

### DISCUSSION THREAD

**Discussion by silverriver** [OpenBMB/CPM-Live#251] [Q&A 问答] [answered]
RT
为了实现双语lm，ant-plus貌似在ant的基础上扩充了英文词表。

那为什么不直接用类似于GPT2的byte-level BPE tokenizer？可以处理所有unicode字符

**Comment by zh-zheng** [comment, marked as answer]
ant已有中文词表，ant-plus想要继续训练的同时支持英文，肯定是使用扩充原有词表的方案，不然就只能重新训练模型了。
